### PR TITLE
account lift cycle management

### PR DESCRIPTION
**Pull request by chaychen2005**

### CODE REVIEW

File: libprecompiled/ChainGovernancePrecompiled.h [hunk 1]
```diff
@@ -30,6 +30,21 @@ class Table;
...
+{
+    AccInvalid = 0,
+    AccAvailable,
+    AccFrozen,
+    AccAddressNonExistent,
+    InvalidAccAddress,
+    AccCount
+};
+
+const std::string ACCOUNT_STATUS_DESC[AccountStatus::AccCount] = {"Invalid",
```
Comment (bxq2011hust) [inline libprecompiled/ChainGovernancePrecompiled.h:43]: if only cpp use it, put these in cop?

File: libprecompiled/Common.h [hunk 2]
```diff
@@ -48,6 +48,10 @@ const std::string SYS_AC_ENABLENUM = "enable_num";
 enum PrecompiledError : int
 {
     // ChainGovernancePrecompiled -52099 ~ -52000
+    CODE_ACCOUNT_FROZEN = -52011,
+    CODE_ACCOUNT_AVAILABLE = -52010,
```
Comment (bxq2011hust) [inline libprecompiled/Common.h:52]:
```suggestion
    CODE_ACCOUNT_ALREADY_AVAILABLE = -52010,
```
add already, may be more clear?

File: libprecompiled/ChainGovernancePrecompiled.cpp [hunk 3]
```diff
@@ -566,3 +586,173 @@ string ChainGovernancePrecompiled::queryCommitteeMembers(
...
+    Table::Ptr table = openTable(context, tableName);
+    if (!table)
+    {
+        return AccountStatus::AccAddressNonExistent;
+    }
+
+    auto codeHashEntries = table->select(storagestate::ACCOUNT_CODE_HASH, table->newCondition());
+    if (toHex(EmptySHA3) != codeHashEntries->get(0)->getField(storagestate::STORAGE_VALUE))
+    {
+        return AccountStatus::InvalidAccAddress;
```
Comment (bxq2011hust) [inline libprecompiled/ChainGovernancePrecompiled.cpp:620]:
```suggestion
        return AccountStatus::InvalidAddress;
```

File: libprecompiled/ChainGovernancePrecompiled.h (hunk 1 above)
Comment (chaychen2005) [reply libprecompiled/ChainGovernancePrecompiled.h]: Done

File: libprecompiled/Common.h (hunk 2 above)
Comment (chaychen2005) [reply libprecompiled/Common.h]: Done

File: libprecompiled/ChainGovernancePrecompiled.cpp (hunk 3 above)
Comment (chaychen2005) [reply libprecompiled/ChainGovernancePrecompiled.cpp]: Done,change to InvalidAccountAddress

**Pull request merged**